acl/src/test/java/org/springframework/security/acls/jdbc/AclClassIdUtilsTest.java

```diff
@@ -120,7 +118,9 @@ public void shouldReturnLongIfTypeClassNotFound() throws SQLException {
 	public void shouldReturnLongIfTypeClassCannotBeConverted() throws SQLException {
 		// given
 		given(resultSet.getString("class_id_type")).willReturn("java.lang.Long");
+		given(conversionService.convert(DEFAULT_IDENTIFIER_AS_STRING, Long.class)).willReturn(new Long(DEFAULT_IDENTIFIER_AS_STRING));
```

Comment:
Really, it feels super smelly that we are mocking this service so that it a) converts Strings to Longs and b) states that it cannot convert Strings to Longs.

We should probably have a separate ticket to clean this up (something like `AclClassIdUtils#convertToLong` should not assume `conversionService` can convert to `Long`). Would you mind adding that?